### ISSUE TITLE
AttributeError: 'module' object has no attribute 'subprocess_info

### ISSUE THREAD

**Issue by HormyAJP** [noonat/sublime-gofmt#4]
Recently upgraded Sublime 3 to 3.0 Build 3143. Since then, every time I save a `.go` file I get an exception:

```
Traceback (most recent call last):
  File "gofmt in /Users/XXXX/Library/Application Support/Sublime Text 3/Installed Packages/Gofmt.sublime-package", line 233, in run_formatter
  File "gofmt in /Users/XXXX/Library/Application Support/Sublime Text 3/Installed Packages/Gofmt.sublime-package", line 144, in __init__
  File "gofmt in /Users/XXXX/Library/Application Support/Sublime Text 3/Installed Packages/Gofmt.sublime-package", line 144, in <listcomp>
  File "gofmt in /Users/XXXX/Library/Application Support/Sublime Text 3/Installed Packages/Gofmt.sublime-package", line 68, in __init__
AttributeError: 'module' object has no attribute 'subprocess_info'
```

Looks like `subprocess_info` is expected to be a attribute of `golangconfig`. Guessing either this component or `golangconfig` changed during the update and broke this.

Haven't looked much further yet I'm afraid (sorry, no time).

**Comment by noonat** [comment]
This appears to be fixed with v0.1.2. I think it may have been caused by the missing dependency.

**Comment by p3lim** [comment]
Getting the same error, just other line numbers.
Windows 10, ST 3175, Go 1.11.0
